### PR TITLE
data:iqrafirdose

### PR DESCRIPTION
**Pull request by iqrafirdose**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/3550"></a>

### CODE REVIEW

**Review by maheshkasabe — DISMISSED** [review]
Looks great

Thanks for the contribution !

File: data/iqrafirdose.json [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+    "avatar": "https://github.com/iqrafirdose.png",
+  "tags": ["Developer","Open Source", "Web dev", "Blogger"],
+  "socials": [
+    { "icon": "FaTwitter", "url": "https://twitter.com/iqra_firdose" },
+    {"icon": "FaLinkedin", "url":"https://www.linkedin.com/in/iqrafirdose/" }
+    { "icon": "FaGithub", "url": "https://github.com/iqrafirdose" },
+    { "icon": "SiHashnode", "url": "https://iqra-firdose.hashnode.dev/" },
+  ],
+
+  "avatar": "https://github.com/iqrafirdose.png",
```
Comment (arpitghura) [inline data/iqrafirdose.json:15]: remove this line as you have already declared.

File: data/iqrafirdose.json [hunk 2]
```diff
@@ -0,0 +1,87 @@
+{
+  "name": "Iqra Firdose",
+  "type": "personal",
+  "displayStatsPublic": true,
+  "bio": " **Open Source** contributor  **Aviyel** ,Hackthoberfest|Web dev Enthisiast | Technical writer | Communities| ",
+    "avatar": "https://github.com/iqrafirdose.png",
+  "tags": ["Developer","Open Source", "Web dev", "Blogger"],
+  "socials": [
+    { "icon": "FaTwitter", "url": "https://twitter.com/iqra_firdose" },
+    {"icon": "FaLinkedin", "url":"https://www.linkedin.com/in/iqrafirdose/" }
```
Comment (arpitghura) [inline data/iqrafirdose.json:10]: Add `;` at the end of `}` to make it proper json

File: data/iqrafirdose.json [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+    { "icon": "FaGithub", "url": "https://github.com/iqrafirdose" },
+    { "icon": "SiHashnode", "url": "https://iqra-firdose.hashnode.dev/" },
+  ],
+
+  "avatar": "https://github.com/iqrafirdose.png",
+  "links": [
+    {
+      "name": "Be a Part of my journey by following me on Twitter",
+      "url": "https://twitter.com/iqra_firdose"
```
Comment (arpitghura) [inline data/iqrafirdose.json:19]: Add `,` at the end.

**Review by arpitghura — CHANGES_REQUESTED** [review]

File: data/iqrafirdose.json (hunk 2 above)
Comment (arpitghura) [reply data/iqrafirdose.json]: Add `,` after all `}`, I have accidentally typed `;`.

File: data/iqrafirdose.json [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+    },
+     
+    {
+      "title": "Started Tech Blog",
+      "date": "October 2022",
+      "icon": "SiHashnode",
+      "color": "blue",
+      "description": "I write blogs on open-source,Web development,my learnings and computer science.",
+      "url": "https://iqra-firdose.hashnode.dev/"
+    }
```
Comment (arpitghura) [inline data/iqrafirdose.json]: Add `,` after `}` to each object

File: data/iqrafirdose.json [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
+  "type": "personal",
+  "displayStatsPublic": true,
+  "bio": " **Open Source** contributor  **Aviyel** ,Hackthoberfest|Web dev Enthisiast | Technical writer | Communities| ",
+    "avatar": "https://github.com/iqrafirdose.png",
+  "tags": ["Developer","Open Source", "Web dev", "Blogger"],
+  "socials": [
+    { "icon": "FaTwitter", "url": "https://twitter.com/iqra_firdose" },
+    {"icon": "FaLinkedin", "url":"https://www.linkedin.com/in/iqrafirdose/" },
+    { "icon": "FaGithub", "url": "https://github.com/iqrafirdose" },
+    { "icon": "SiHashnode", "url": "https://iqra-firdose.hashnode.dev/" },
```
Comment (Pradumnasaraf) [inline data/iqrafirdose.json:12]:
```suggestion
    { "icon": "SiHashnode", "url": "https://iqra-firdose.hashnode.dev/" }
```

File: data/iqrafirdose.json (hunk 4 above)
Comment (Pradumnasaraf) [inline data/iqrafirdose.json:67]:
```suggestion
    },
```

File: data/iqrafirdose.json [hunk 6]
```diff
@@ -0,0 +1,85 @@
...
+      "url": "https://iqra-firdose.hashnode.dev/"
+    }
+    {
+      "title": "Develevate Sliver Winner",
+      "date": "December 2022",
+      "icon": "FiLink",
+      "color": "blue",
+      "description": "Develevate is organised by Showwcase",
+      "url": "https://www.showwcase.com/show/25223/dev-elevate-silver-winners"
+    }
```
Comment (Pradumnasaraf) [inline data/iqrafirdose.json:75]:
```suggestion
    },
```

**Review by Pradumnasaraf — CHANGES_REQUESTED** [review]
Hey,

The file still needs some JSON syntax correction. I have requested the change. If you need any help feel free to ask

**Comment by iqrafirdose** [comment]
> Hey,
> 
> The file still needs some JSON syntax correction. I have requested the change. If you need any help feel free to ask

I did the changes that you suggested and it again fails in JSON syntax.

File: data/iqrafirdose.json [hunk 7]
```diff
@@ -0,0 +1,85 @@
+{
+  "name": "Iqra Firdose",
+  "type": "personal",
+  "displayStatsPublic": true,
+  "bio": " **Open Source** contributor  **Aviyel** ,Hackthoberfest|Web dev Enthisiast | Technical writer | Communities| ",
+    "avatar": "https://github.com/iqrafirdose.png",
+  "tags": ["Developer","Open Source", "Web dev", "Blogger"],
+  "socials": [
+    { "icon": "FaTwitter", "url": "https://twitter.com/iqra_firdose" }
```
Comment (arpitghura) [inline data/iqrafirdose.json:9]: Add `,` at the end 
```suggestion
    { "icon": "FaTwitter", "url": "https://twitter.com/iqra_firdose" },
```

File: data/iqrafirdose.json [hunk 8]
```diff
@@ -0,0 +1,85 @@
...
+    {"icon": "FaLinkedin", "url":"https://www.linkedin.com/in/iqrafirdose/" }
```
Comment (arpitghura) [inline data/iqrafirdose.json:10]:
```suggestion
    {"icon": "FaLinkedin", "url":"https://www.linkedin.com/in/iqrafirdose/" },
```

File: data/iqrafirdose.json [hunk 9]
```diff
@@ -0,0 +1,85 @@
...
+    { "icon": "FaGithub", "url": "https://github.com/iqrafirdose" }
```
Comment (arpitghura) [inline data/iqrafirdose.json:11]:
```suggestion
    { "icon": "FaGithub", "url": "https://github.com/iqrafirdose" },
```

**Review by arpitghura — CHANGES_REQUESTED** [review]

File: data/iqrafirdose.json (hunk 5 above)
Comment (arpitghura) [reply data/iqrafirdose.json]: I think the comma should be there as these are objects in an array which needs to be seperated by comma

File: data/iqrafirdose.json [hunk 10]
```diff
@@ -0,0 +1,85 @@
...
+      "icon": "SiHashnode"
+    },
+    {
+      "name": "Blogs at Showwcase",
+      "url": "https://www.showwcase.com/iqrafirdose639401?tab=1",
+      "icon": "FaLink"
+    }
+  ]
+   "milestones": [
+
```
Comment (arpitghura) [inline data/iqrafirdose.json:42]: Remove this extra line break

File: data/iqrafirdose.json [hunk 11]
```diff
@@ -0,0 +1,85 @@
...
+    },
+      {
+      "title": "Hackthoberfest with Aviyel",
+      "date": "October 2022",
+      "icon": "FaGithub",
+      "color": "red",
+      "description": "Become a top creator of Hackthoberfest with Aviyel",
+      "url": "https://aviyel.com/@iqrafirdose"
+    },
+
```
Comment (arpitghura) [inline data/iqrafirdose.json:59]: Remove this extra line break

**Review by arpitghura — APPROVED** [review]
Remove these lines and rest all things good.

**Comment by Pradumnasaraf** [comment]
> Remove these lines and rest all things good.

There is no need to request for removing extra lines. Action will do after the merge. Thank you

**Comment by iqrafirdose** [comment]
I hope now it's works

**Review by Pradumnasaraf — APPROVED** [review]
Looks good to merge. Thank you for your contribution! :tada: 

<p align="center"></p>

**Comment by iqrafirdose** [comment]
Hello, When I can able to see my Eddiehub linkfree?

**Comment by Pradumnasaraf** [comment]
> Hello, When I can able to see my Eddiehub linkfree?

Its already live at https://linkfree.eddiehub.io/iqrafirdose. If you are still unable to see it, do let's know.

**Pull request merged**